_query-dsl/query-dsl/minimum-should-match.md

```diff
@@ -0,0 +1,450 @@
+Negative percentage | `-30%` | A document can have this percentage of the total number of optional clauses that do not match. The number of clauses a document is allowed not to match is rounded down to the nearest integer.
+Combination | `2<75%` | Expression in the `n<p%` format. If the number of optional clauses is less than or equal to `n`, the document must match all optional clauses. If the number of optional clauses is greater than `n`, then the document must match the `p` percentage of optional clauses.
+Multiple combinations | `3<-1 5<50%` | More than one combination separated by a space. Each condition applies to the number of optional clauses that is greater than the number on the left of the `<` sign. In this example, if there are 3 or less optional clauses, the document must match all of them. If there are 4 or 5 optional clauses, the document must match all but 1 of them. If there are 6 or more optional clauses, the document must match 50% of them.
+
+Let `n` be the number of optional clauses a document must match. When `n` is calculated as a percentage, if `n` is less than 1, then 1 is used. If `n` is greater than the number of optional clauses, the number of optional clauses is used.
+{: .note}
+
+
+## Using the parameter in Boolean queries
+
+A [Boolean query]({{site.url}}{{site.baseurl}}/) lists optional clauses in the `should` clause and required clauses in the `must` clause. Optionally, it can contain a `filter` clause to filter results.
+
+Consider an example index with the following five documents:
+
+```json
+PUT testindex/_doc/1
+{
+  "text": "one OpenSearch"
+}
+```
+{% include copy-curl.html %}
+
+```json
+PUT testindex/_doc/2
+{
+  "text": "one two OpenSearch"
+}
+```
+{% include copy-curl.html %}
+
+```json
+PUT testindex/_doc/3
+{
+  "text": "one two three OpenSearch"
+}
+```
+{% include copy-curl.html %}
+
+```json
+PUT testindex/_doc/4
+{
+  "text": "one two three four OpenSearch"
+}
+```
+{% include copy-curl.html %}
+
+```json
+PUT testindex/_doc/5
+{
+  "text": "OpenSearch"
+}
+```
+{% include copy-curl.html %}
+
+The following query contains four optional clauses:
+
+```json
+GET testindex/_search
+{
+  "query": {
+    "bool": {
+      "must": [
+        {
+          "match": {
+            "text": "OpenSearch"
+          }
+        }
+      ], 
+      "should": [
+        {
+          "match": {
+            "text": "one"
+          }
+        },
+        {
+          "match": {
+            "text": "two"
+          }
+        },
+        {
+          "match": {
+            "text": "three"
+          }
+        },
+        {
+          "match": {
+            "text": "four"
+          }
+        }
+      ],
+      "minimum_should_match": "80%"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Because `minimum_should_match` is `80%`, the number of optional clauses to match is calculated as 4 &middot; 0.8 = 3.2 and then rounded down to 3. Therefore, the results contain documents that match at least 3 clauses:
```

Comment:
```suggestion
Because `minimum_should_match` is `80%`, the number of optional clauses to match is calculated as 4 &middot; 0.8 = 3.2 and then rounded down to 3. Therefore, the results contain documents that match at least three clauses:
```